resources/master_term_bank_en.csv

```diff
@@ -59,4 +59,6 @@ application/json,block,"The term block refers to data representing a set of tran
 application/json,centralized,"The term centralized is often used to describe a good or service tied to a single entity (or small group) for which many users depend upon. A centralized good or service often consolidates control, authority and access to the small group of participants that operate it; however, that often makes the small group in control a single point of failure. Much of the web2 landscape is dominated by centralized systems and platforms in contrast to the web3 movement that is leaning towards a more decentralized ecosystem.",en,GLOSSETA-PROD,general
-application/json,sharding,"Sharding is typically regarded as a database optimization technique where tables are partitioned (into shards) and distributed among multiple database instances to increase overall throughput. This technique can also be applied to blockchains where nodes in a given network hold parts of the overall chain so that each shard can work in parallel to reach consensus in a more load balanced manner. This overall optimization in the consensus mechanism of the chain theoretically leads to an overall increase in network transaction speeds.",en,GLOSSETA-PROD,general
\ No newline at end of file
+application/json,sharding,"Sharding is typically regarded as a database optimization technique where tables are partitioned (into shards) and distributed among multiple database instances to increase overall throughput. This technique can also be applied to blockchains where nodes in a given network hold parts of the overall chain so that each shard can work in parallel to reach consensus in a more load balanced manner. This overall optimization in the consensus mechanism of the chain theoretically leads to an overall increase in network transaction speeds.",en,GLOSSETA-PROD,general
+application/json,airdrops,"Airdrops are essentially when creators of a project send free coins or tokens to the wallets of users of their products to raise awareness around the project.",en,GLOSSETA-PROD,general
+application/json,bearish,"this refers to holding a pessimistic view of a market or asset’s value, If you are bearish on a certain cryptocurrency, you believe its value will decrease over time",en,GLOSSETA-PROD,general
```

Comment:
Content is great overall, just a few tweaks:

`This term refers to holding a pessimistic view of a market or asset’s value. For example, if you are bearish on a certain cryptocurrency, you believe its value will decrease over time.`